rd_workshop/styles/blocks/main.scss

```diff
@@ -0,0 +1,404 @@
+            z-index: 10;
+            padding-top: 28px;
+
+            h3 {
+                color: $main-color;
+                font-size: 18px;
+                font-weight: 700;
+                line-height: 22px;
+            }
+
+            p {
+                color: $footer-color;
+                text-align: center;
+                font-size: 14px;
+                font-weight: 400;
+                line-height: 22px;
+                padding-top: 14px;
+                height: 66px;
+                width: 180px;
+                margin: 0 auto;
+            }
+        }
+    }
+    .button {
+        top: 2185px;
+        left: 857px;
+        position: absolute;
+
+        .button-help-services {
+            border-radius: 10px;
+            width: 192px;
+            height: 62px;
+            background-color: $main-color;
+            border: 1px solid $secondary-color;
+            color: $secondary-color;
+            text-align: center;
+            text-decoration: none;
+            display: inline-block;
+            font-size: 20px;
+        
+            &:hover {
+                background-color: $footer-color;
+                box-shadow: 0 4px 8px 0 rgba(245, 239, 239, 0.2), 0 6px 20px 0 $secondary-color;
+            }      
+        }
+    }
+
+    .testymonials {
+        width: 1903px;
+        height: 718px;
+        background-color: $main-color;
+
+        @media screen and (max-width: 1600px) {
+            & {
+                width: 1783px;
+            }
+        }
+        @media screen and (max-width: 1450px) {
+            & {
+                width: 1380px;
+                height: 1250px;
+                padding-top: 110px;
+                .container {
+                    .testymonials-container-none {
+                        flex-direction: column;
+                        align-items: center;
+                    }
+                }
+            }
+        }
+    
+        h2 {
+            color: $secondary-color;
+            font-size: 58px;
+            text-align: center;
+            font-style: normal;
+            font-weight: bold;
+            padding-top: 30px;
+            padding-bottom: 30px;
+        }
+
+        &-text-content {
+          display: flex;
+          flex-direction: column;
+          .stars {
+            display: flex;
+            padding-left: 14px;
+            padding-bottom: 30px;
+          }
+
+          h3 {
+            padding-top: 25px;
+            padding-left: 14px;
+            color: $main-color;
+            font-size: 18px;
+            font-weight: 700;
+            line-height: 22px;
+          }
+
+          p {
+            color: $footer-color;
+            font-size: 14px;
+            font-weight: 400;
+            line-height: 22px;
+            width: 423px;
+            height: 110px;
+            padding-top: 8px;
+            padding-left: 14px;
+          }
+        }
+
+        .container {
```

Comment:
please, never NEVER assign any styles to Bootstrap classes, that's very huge mistake